packages/components/forms/src/Radio/README.mdx

```diff
@@ -69,6 +69,27 @@ We also provide the `Radio.Group` component that helps when using multiple radio
 
 ```
 
+## Content guidelines
+
+- Use direct, succinct copy that helps a user to successfully complete a form for example “Dinner menu (select 1 of 3 options)“
+- Do not use punctuation for labels for example “Vegetarian“ instead of “Vegetarian.“
+- Use sentence style caps in which only the first word of a sentence or term is capitalized for example “Dairy free“ instead of “Dairy Free“
```

Comment:
Nit picking 🙂 

```suggestion
- Use direct, succinct copy that helps a user to successfully complete a form, for example, “Dinner menu (select 1 of 3 options)“
- Do not use punctuation for labels, for example, “Vegetarian“ instead of “Vegetarian.“
- Use sentence style caps in which only the first word of a sentence or term is capitalized, for example, “Dairy free“ instead of “Dairy Free“
```